packages/dds/tree/.mocharc.customBenchmarks.cjs

```diff
@@ -29,5 +29,5 @@ module.exports = {
 	"recursive": true,
 	"reporter": "@fluid-tools/benchmark/dist/MochaReporter.js",
 	"reporterOptions": ["reportDir=.customBenchmarksOutput/"],
-	"spec": ["lib/test/**/*.*js", "--perfMode"],
+	"spec": ["lib/test/**/*.*js"],
```

Comment:
For reference: @jikim-msft and I discussed this and realized that `perfMode` is not necessary for custom benchmarks, since we don't run different code paths like we do for execution time and memory usage benchmarks (run tests once as unit tests when not in perf mode, vs iterating them when running in perf mode; custom benchmarks, at least as designed today, should be run once in either case).